target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -91,6 +91,12 @@ ath79_setup_interfaces()
 	ocedo,ursus)
 		ucidef_set_interface_lan "eth0 eth1"
 		;;
+	atheros,db120)
+		ucidef_add_switch "switch0" \
```

Comment:
Does it really work automagically with two switches, or do we need a
`ucidef_set_interfaces_lan_wan "eth0.1 eth1.1" "eth0.2"`
here?

Reply:
It does work, LAN is bridged together as the two ethX.1 interfaces, and eth0.2 is set as the WAN interface.

Reply:
Nice, our scripts are smarter than I thought :-)